docs/extend/plugins/urls.md

```diff
@@ -8,13 +8,35 @@ Use the class constant `URLS` for a array of URLs that should be added to InvenT
 
 The array has to contain valid URL patterns as defined in the [django documentation](https://docs.djangoproject.com/en/stable/topics/http/urls/).
 
+Minimal sample:
+
```

Comment:
Suggestion: All samples are minimal; Full working examples are provided in the main code base so this is redundant.
```suggestion
```